### ISSUE TITLE
Change Cassandra replication-strategy-class and replication-strategy-options to GLOBAL_OFFLINE

### ISSUE THREAD

**Issue by twilmes** [thinkaurelius/titan#1285]
Right now storage.cassandra.replication-factor is set to GLOBAL_OFFLINE, but replication-strategy-class and replication-strategy-options are set to FIXED.  I think these could be safely updated to GLOBAL_OFFLINE to support changes in the Cassandra cluster topology, eg. moving from SimpleStrategy to NetworkTopologyStrategy or possibly adding a new DC to an existing cluster.

**Comment by boliza** [comment]
:+1: ,but we change the replication-factor in C\* cluster。

**Comment by twilmes** [comment]
Yes, I should have been a bit more clear.  After Titan keyspace initialization, you would manage these things with cqlsh.  The issue that I noticed is that if you do update the config to match your cqlsh changes (maybe just for clarity purposes), you get an error on connect due to the mismatch between your config and what Titan has stored as those FIXED property values in the system_properties CF.  This ticket is more to bring these properties inline with the plain old replication-factor property if folks think that makes sense.  Overall, I'd say it is a low priority though.
